### ISSUE TITLE
New term : "Tubulopathy"

### ISSUE THREAD

**Issue by AliceChoury** [obophenotype/human-phenotype-ontology#4329]
_For new term requests, please provide us with the following information:_
### 1. Preferred term label
Tubulopathy

### 2. Synonyms
Renal tubulopathy, Tubule dysfunction, Tubular damage, Renal tubule defect, Tubular defect

### 3. Textual definition (should be understandable even for non-specialists, please include a PubMed ID for relevant articles providing additional information about the suggestion)

Dysfunction of the tubule for the urine transport.

### 4. Parent term (use [HPO app](https://hpo.jax.org/app/), [HPO Browser](http://compbio.charite.de/hpoweb/showterm?id=HP:0000118) or [OLS](http://www.ebi.ac.uk/ols/ontologies/hp)) 

Abnormality of the renal tubule :
[HPO app](https://hpo.jax.org/app/browse/term/HP:0000091)

### 4.b Children term
Tubulopathy proximal : 
[HPO app](https://hpo.jax.org/app/browse/term/HP:0000114)
Tubulopathy distal
Tubulopathy of the collecting canal

**Comment by drseb** [comment]
Thanks for your request. If I may ask: have you seen the class http://compbio.charite.de/hpoweb/showterm?id=HP:0000118#id=HP:0000124 ?

**Comment by AliceChoury** [comment]
Yes and no. 
Because Tubulointerstitial abnormality, Renotubular dysgenesis, Acute tubular necrosis, Tubular basement membrane disintegration and Tubular atrophy are not a tubulopathy.

**Comment by pnrobinson** [comment]
I think it is necessary to think of both functional and morphological abnormalities. 
We are going to be having a large-scale kidney workshop this year and will be working on refining this in a systematic way. The above term currently refers to functional abnormalities of the tubule. This term refers to morphological abnormalities
https://hpo.jax.org/app/browse/term/HP:0000091
I will take a look at the Renal tubular dysfunction term -- it seems to require a btter definition.
Please note that HPO tries to separate structure and function, and the term "Tubulopathy" seems to include both aspects to some extent (it is not really strictly defined)

**Comment by pnrobinson** [comment]
I have updated the definitions. I think that the term "Renal tubular dysfunction" is what you mean by tubulopathy. I have relabled Abnormality of the renal tubule to Abnormal renal tubule morphology. This section will need additional work and new term requests are welcome. We are having an initial renal workshop in about two months, and will be working on the renal section throughout this year.
